.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -14,6 +14,8 @@ assignees: ''
 * apisix version (cmd: `apisix version`):
 * OS: (cmd: `uname -a`)
 * OpenResty / Nginx version: (cmd: `nginx -V` or `openresty -V`)
+* etcd version, if have: (cmd: `curl http://$etcd_host:$etcdport/version`)
+* apisix-dashboard version, if have:
```

Comment:
I think maybe we can guide users to print the output of the control API endpoint of server-info plugin (if the enabled).

Reply:
Updated.